test/recipes/controls/aws_parallelcluster_install/chrony_spec.rb

```diff
@@ -0,0 +1,44 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'chrony' do
+  title 'Test chrony installation and configuration'
+
+  describe package('chrony') do
+    it { should be_installed }
+  end unless os_properties.redhat_ubi?
+
+  describe package('ntp') do
```

Comment:
we can split the controls, have one for `ntp_uninstalled` and another for chrony specific